Mediakit 1.1.2: Cannot build on MacOS - `ld: framework not found Mpv`

I tried updating media_kit to the newest version 1.1.2 and am now hitting this error while building on MacOS on M1 pro.
```
Connected devices:
macOS (desktop) • macos • darwin-arm64 • macOS 13.4.1 22F770820d darwin-arm64

Checking for wireless devices...

[1]: macOS (macos)
Please choose one (or "q" to quit): 1
Launching lib/main.dart on macOS in release mode...
2023-08-06 23:10:54.816 xcodebuild[2805:6788141] DVTCoreDeviceEnabledState: DVTCoreDeviceEnabledState_Disabled set via user default (DVTEnableCoreDevice=disabled)
--- xcodebuild: WARNING: Using the first of multiple matching destinations:
{ platform:macOS, arch:arm64, id:00006000-000265022223401E }
{ platform:macOS, arch:x86_64, id:00006000-000265022223401E }
ld: warning: directory not found for option '-F/Users/dario/dev/DaKanji/DaKanjiApplication/macos/Pods/../Flutter/ephemeral/.symlinks/plugins/media_kit_libs_macos_video/macos/Frameworks/.symlinks/mpv/macos'
ld: framework not found Mpv
clang: error: linker command failed with exit code 1 (use -v to see invocation)
ld: warning: directory not found for option '-F/Users/dario/dev/DaKanji/DaKanjiApplication/macos/Pods/../Flutter/ephemeral/.symlinks/plugins/media_kit_libs_macos_video/macos/Frameworks/.symlinks/mpv/macos'
ld: framework not found Mpv
clang: error: linker command failed with exit code 1 (use -v to see invocation)
warning: Run script build phase 'Run Script' will be run during every build because it does not specify any outputs. To address this warning, either add output dependencies to the script phase, or configure it to run in every build by unchecking "Based on dependency analysis" in the script phase. (in target 'Flutter Assemble' from project 'Runner')
warning: Run script build phase 'Run Script' will be run during every build because it does not specify any outputs. To address this warning, either add output dependencies to the script phase, or configure it to run in every build by unchecking "Based on dependency analysis" in the script phase. (in target 'Runner' from project 'Runner')
** BUILD FAILED **

Building macOS application...                                           
Exception: Build process failed
#0      throwToolExit (package:flutter_tools/src/base/common.dart:10:3)
#1      buildMacOS (package:flutter_tools/src/macos/build_macos.dart:141:5)
<asynchronous suspension>
#2      MacOSDevice.buildForDevice (package:flutter_tools/src/macos/macos_device.dart:72:5)
<asynchronous suspension>
#3      DesktopDevice.startApp (package:flutter_tools/src/desktop_device.dart:119:7)
<asynchronous suspension>
#4      FlutterDevice.runCold (package:flutter_tools/src/resident_runner.dart:524:33)
<asynchronous suspension>
#5      ColdRunner.run (package:flutter_tools/src/run_cold.dart:57:28)
<asynchronous suspension>
#6      RunCommand.runCommand (package:flutter_tools/src/commands/run.dart:750:27)
<asynchronous suspension>
#7      FlutterCommand.run.<anonymous closure> (package:flutter_tools/src/runner/flutter_command.dart:1336:27)
<asynchronous suspension>
#8      AppContext.run.<anonymous closure> (package:flutter_tools/src/base/context.dart:150:19)
<asynchronous suspension>
#9      CommandRunner.runCommand (package:args/command_runner.dart:212:13)
<asynchronous suspension>
#10     FlutterCommandRunner.runCommand.<anonymous closure> (package:flutter_tools/src/runner/flutter_command_runner.dart:296:9)
<asynchronous suspension>
#11     AppContext.run.<anonymous closure> (package:flutter_tools/src/base/context.dart:150:19)
<asynchronous suspension>
#12     FlutterCommandRunner.runCommand (package:flutter_tools/src/runner/flutter_command_runner.dart:244:5)
<asynchronous suspension>
#13     run.<anonymous closure>.<anonymous closure> (package:flutter_tools/runner.dart:91:9)
<asynchronous suspension>
#14     AppContext.run.<anonymous closure> (package:flutter_tools/src/base/context.dart:150:19)
<asynchronous suspension>
#15     main (package:flutter_tools/executable.dart:91:3)
<asynchronous suspension>
```

1.1.0 throws this:
``` Connected devices:
macOS (desktop) • macos • darwin-arm64 • macOS 13.4.1 22F770820d darwin-arm64

No wireless devices were found.

[1]: macOS (macos)
Please choose one (or "q" to quit): 1
Launching lib/main.dart on macOS in debug mode...
Running pod install...                                           2,907ms
2023-08-06 23:02:49.034 xcodebuild[175:6773776] DVTCoreDeviceEnabledState: DVTCoreDeviceEnabledState_Disabled set via user default (DVTEnableCoreDevice=disabled)
--- xcodebuild: WARNING: Using the first of multiple matching destinations:
{ platform:macOS, arch:arm64, id:00006000-000265022223401E }
{ platform:macOS, arch:x86_64, id:00006000-000265022223401E }
ld: warning: directory not found for option '-F/Users/dario/dev/DaKanji/DaKanjiApplication/macos/Pods/../Flutter/ephemeral/.symlinks/plugins/media_kit_libs_macos_video/macos/Frameworks/.symlinks/mpv/macos'
ld: framework not found Mpv
clang: error: linker command failed with exit code 1 (use -v to see invocation)
warning: Run script build phase 'Run Script' will be run during every build because it does not specify any outputs. To address this warning, either add output dependencies to the script phase, or configure it to run in every build by unchecking "Based on dependency analysis" in the script phase. (in target 'Runner' from project 'Runner')
warning: Run script build phase 'Run Script' will be run during every build because it does not specify any outputs. To address this warning, either add output dependencies to the script phase, or configure it to run in every build by unchecking "Based on dependency analysis" in the script phase. (in target 'Flutter Assemble' from project 'Runner')
** BUILD FAILED **

Building macOS application...                                           
Exception: Build process failed
```

I am using the default dependencies from the README

``` yaml
  #media_kit: android, iOS, macos, windows, linux
  media_kit: ^1.1.2                              # Primary package.
  
  media_kit_video: ^1.1.2                        # For video rendering.
  
  media_kit_native_event_loop: ^1.0.7            # Support for higher number of concurrent instances & better performance.
  
  media_kit_libs_android_video: ^1.3.0           # Android package for video native libraries.
  media_kit_libs_ios_video: ^1.1.0               # iOS package for video native libraries.
  media_kit_libs_macos_video: ^1.1.0             # macOS package for video native libraries.
  media_kit_libs_windows_video: ^1.0.5           # Windows package for video native libraries.
  media_kit_libs_linux: ^1.1.0                   # GNU/Linux dependency package.
```

I tried
```
pod install
pod update
flutter clean
```
and all of them seem not to help.

Sorry if I am missing something obvious.

--- a/README.md
+++ b/README.md
@@ -53,8 +53,8 @@ dependencies:
   media_kit_native_event_loop: ^1.0.7            # Support for higher number of concurrent instances & better performance.
   
   media_kit_libs_android_video: ^1.3.0           # Android package for video native libraries.
-  media_kit_libs_ios_video: ^1.1.0               # iOS package for video native libraries.
-  media_kit_libs_macos_video: ^1.1.0             # macOS package for video native libraries.
+  media_kit_libs_ios_video: ^1.1.1               # iOS package for video native libraries.
+  media_kit_libs_macos_video: ^1.1.1             # macOS package for video native libraries.
   media_kit_libs_windows_video: ^1.0.5           # Windows package for video native libraries.
   media_kit_libs_linux: ^1.1.0                   # GNU/Linux dependency package.
 ```
@@ -68,8 +68,8 @@ dependencies:
   media_kit_native_event_loop: ^1.0.7            # Support for higher number of concurrent instances & better performance.
   
   media_kit_libs_android_audio: ^1.3.0           # Android package for audio native libraries.
-  media_kit_libs_ios_audio: ^1.1.0               # iOS package for audio native libraries.
-  media_kit_libs_macos_audio: ^1.1.0             # macOS package for audio native libraries.
+  media_kit_libs_ios_audio: ^1.1.1               # iOS package for audio native libraries.
+  media_kit_libs_macos_audio: ^1.1.1             # macOS package for audio native libraries.
   media_kit_libs_windows_audio: ^1.0.5           # Windows package for audio native libraries.
   media_kit_libs_linux: ^1.1.0                   # GNU/Linux dependency package.
 ```

--- a/libs/ios/media_kit_libs_ios_audio/CHANGELOG.md
+++ b/libs/ios/media_kit_libs_ios_audio/CHANGELOG.md
@@ -1,3 +1,7 @@
+## 1.1.1
+
+- fix: fix CRLF line terminator added by Flutter packaging (#338)
+
 ## 1.1.0
 
 - build: bump libs to default flavor

--- a/libs/ios/media_kit_libs_ios_audio/example/README.md
+++ b/libs/ios/media_kit_libs_ios_audio/example/README.md
@@ -4,7 +4,7 @@ Add in your `pubspec.yaml`:
 
 ```yaml
 dependencies:
-  media_kit_libs_ios_audio: ^1.1.0
+  media_kit_libs_ios_audio: ^1.1.1
 ```
 
 This will automatically bundle required shared libraries for audio playback on iOS.

--- a/libs/ios/media_kit_libs_ios_audio/ios/Makefile
+++ b/libs/ios/media_kit_libs_ios_audio/ios/Makefile
@@ -26,6 +26,7 @@ Frameworks/*.xcframework: .cache/xcframeworks/libmpv-xcframeworks-ios-universal.
 Frameworks/.symlinks: Frameworks/*.xcframework
 	rm -rf Frameworks/.symlinks
 	mkdir -p Frameworks/.symlinks/mpv
+	sed -i '' 's/\r$$//g' create_framework_symlinks.sh # remove CRLF line terminator added by Flutter packaging (https://github.com/media-kit/media-kit/issues/338)
 	sh create_framework_symlinks.sh Frameworks/Mpv.xcframework Frameworks/.symlinks/mpv
 
 .PHONY: clean

--- a/libs/ios/media_kit_libs_ios_audio/pubspec.yaml
+++ b/libs/ios/media_kit_libs_ios_audio/pubspec.yaml
@@ -1,6 +1,6 @@
 name: media_kit_libs_ios_audio
 description: iOS package providing audio native libraries for package:media_kit.
-version: 1.1.0
+version: 1.1.1
 homepage: https://github.com/media-kit/media-kit.git
 repository: https://github.com/media-kit/media-kit.git
 funding:

--- a/libs/ios/media_kit_libs_ios_video/CHANGELOG.md
+++ b/libs/ios/media_kit_libs_ios_video/CHANGELOG.md
@@ -1,3 +1,7 @@
+## 1.1.1
+
+- fix: fix CRLF line terminator added by Flutter packaging (#338)
+
 ## 1.1.0
 
 - build: bump libs to default flavor

--- a/libs/ios/media_kit_libs_ios_video/example/README.md
+++ b/libs/ios/media_kit_libs_ios_video/example/README.md
@@ -4,7 +4,7 @@ Add in your `pubspec.yaml`:
 
 ```yaml
 dependencies:
-  media_kit_libs_ios_video: ^1.1.0
+  media_kit_libs_ios_video: ^1.1.1
 ```
 
 This will automatically bundle required shared libraries for video (& audio) playback on iOS.

--- a/libs/ios/media_kit_libs_ios_video/ios/Makefile
+++ b/libs/ios/media_kit_libs_ios_video/ios/Makefile
@@ -26,6 +26,7 @@ Frameworks/*.xcframework: .cache/xcframeworks/libmpv-xcframeworks-ios-universal.
 Frameworks/.symlinks: Frameworks/*.xcframework
 	rm -rf Frameworks/.symlinks
 	mkdir -p Frameworks/.symlinks/mpv
+	sed -i '' 's/\r$$//g' create_framework_symlinks.sh # remove CRLF line terminator added by Flutter packaging (https://github.com/media-kit/media-kit/issues/338)
 	sh create_framework_symlinks.sh Frameworks/Mpv.xcframework Frameworks/.symlinks/mpv
 
 .PHONY: clean

--- a/libs/ios/media_kit_libs_ios_video/pubspec.yaml
+++ b/libs/ios/media_kit_libs_ios_video/pubspec.yaml
@@ -1,6 +1,6 @@
 name: media_kit_libs_ios_video
 description: iOS package providing video (& audio) native libraries for package:media_kit.
-version: 1.1.0
+version: 1.1.1
 homepage: https://github.com/media-kit/media-kit.git
 repository: https://github.com/media-kit/media-kit.git
 funding:

--- a/libs/macos/media_kit_libs_macos_audio/CHANGELOG.md
+++ b/libs/macos/media_kit_libs_macos_audio/CHANGELOG.md
@@ -1,3 +1,7 @@
+## 1.1.1
+
+- fix: fix CRLF line terminator added by Flutter packaging (#338)
+
 ## 1.1.0
 
 - build: bump libs to default flavor

--- a/libs/macos/media_kit_libs_macos_audio/example/README.md
+++ b/libs/macos/media_kit_libs_macos_audio/example/README.md
@@ -4,7 +4,7 @@ Add in your `pubspec.yaml`:
 
 ```yaml
 dependencies:
-  media_kit_libs_macos_audio: ^1.1.0
+  media_kit_libs_macos_audio: ^1.1.1
 ```
 
 This will automatically bundle required shared libraries for audio playback on macOS.

--- a/libs/macos/media_kit_libs_macos_audio/macos/Makefile
+++ b/libs/macos/media_kit_libs_macos_audio/macos/Makefile
@@ -26,6 +26,7 @@ Frameworks/*.xcframework: .cache/xcframeworks/libmpv-xcframeworks-macos-universa
 Frameworks/.symlinks: Frameworks/*.xcframework
 	rm -rf Frameworks/.symlinks
 	mkdir -p Frameworks/.symlinks/mpv
+	sed -i '' 's/\r$$//g' create_framework_symlinks.sh # remove CRLF line terminator added by Flutter packaging (https://github.com/media-kit/media-kit/issues/338)
 	sh create_framework_symlinks.sh Frameworks/Mpv.xcframework Frameworks/.symlinks/mpv
 
 .PHONY: clean

--- a/libs/macos/media_kit_libs_macos_audio/pubspec.yaml
+++ b/libs/macos/media_kit_libs_macos_audio/pubspec.yaml
@@ -1,6 +1,6 @@
 name: media_kit_libs_macos_audio
 description: macOS package providing audio native libraries for package:media_kit.
-version: 1.1.0
+version: 1.1.1
 homepage: https://github.com/media-kit/media-kit.git
 repository: https://github.com/media-kit/media-kit.git
 funding:

--- a/libs/macos/media_kit_libs_macos_video/CHANGELOG.md
+++ b/libs/macos/media_kit_libs_macos_video/CHANGELOG.md
@@ -1,3 +1,7 @@
+## 1.1.1
+
+- fix: fix CRLF line terminator added by Flutter packaging (#338)
+
 ## 1.1.0
 
 - build: bump libs to default flavor

--- a/libs/macos/media_kit_libs_macos_video/example/README.md
+++ b/libs/macos/media_kit_libs_macos_video/example/README.md
@@ -4,7 +4,7 @@ Add in your `pubspec.yaml`:
 
 ```yaml
 dependencies:
-  media_kit_libs_macos_video: ^1.1.0
+  media_kit_libs_macos_video: ^1.1.1
 ```
 
 This will automatically bundle required shared libraries for video (& audio) playback on macOS.

--- a/libs/macos/media_kit_libs_macos_video/macos/Makefile
+++ b/libs/macos/media_kit_libs_macos_video/macos/Makefile
@@ -26,6 +26,7 @@ Frameworks/*.xcframework: .cache/xcframeworks/libmpv-xcframeworks-macos-universa
 Frameworks/.symlinks: Frameworks/*.xcframework
 	rm -rf Frameworks/.symlinks
 	mkdir -p Frameworks/.symlinks/mpv
+	sed -i '' 's/\r$$//g' create_framework_symlinks.sh # remove CRLF line terminator added by Flutter packaging (https://github.com/media-kit/media-kit/issues/338)
 	sh create_framework_symlinks.sh Frameworks/Mpv.xcframework Frameworks/.symlinks/mpv
 
 .PHONY: clean

--- a/libs/macos/media_kit_libs_macos_video/pubspec.yaml
+++ b/libs/macos/media_kit_libs_macos_video/pubspec.yaml
@@ -1,6 +1,6 @@
 name: media_kit_libs_macos_video
 description: macOS package providing video (& audio) native libraries for package:media_kit.
-version: 1.1.0
+version: 1.1.1
 homepage: https://github.com/media-kit/media-kit.git
 repository: https://github.com/media-kit/media-kit.git
 funding:

--- a/media_kit/CHANGELOG.md
+++ b/media_kit/CHANGELOG.md
@@ -1,3 +1,11 @@
+## 1.1.2+1
+
+- docs: document updated
+  - `media_kit_libs_ios_audio`
+  - `media_kit_libs_ios_video`
+  - `media_kit_libs_macos_audio`
+  - `media_kit_libs_macos_video`
+
 ## 1.1.2
 
 - feat: export `PlayerState` & `PlayerStream`

--- a/media_kit/README.md
+++ b/media_kit/README.md
@@ -45,8 +45,8 @@ dependencies:
   media_kit_native_event_loop: ^1.0.7            # Support for higher number of concurrent instances & better performance.
   
   media_kit_libs_android_video: ^1.3.0           # Android package for video native libraries.
-  media_kit_libs_ios_video: ^1.1.0               # iOS package for video native libraries.
-  media_kit_libs_macos_video: ^1.1.0             # macOS package for video native libraries.
+  media_kit_libs_ios_video: ^1.1.1               # iOS package for video native libraries.
+  media_kit_libs_macos_video: ^1.1.1             # macOS package for video native libraries.
   media_kit_libs_windows_video: ^1.0.5           # Windows package for video native libraries.
   media_kit_libs_linux: ^1.1.0                   # GNU/Linux dependency package.
 ```
@@ -60,8 +60,8 @@ dependencies:
   media_kit_native_event_loop: ^1.0.7            # Support for higher number of concurrent instances & better performance.
   
   media_kit_libs_android_audio: ^1.3.0           # Android package for audio native libraries.
-  media_kit_libs_ios_audio: ^1.1.0               # iOS package for audio native libraries.
-  media_kit_libs_macos_audio: ^1.1.0             # macOS package for audio native libraries.
+  media_kit_libs_ios_audio: ^1.1.1               # iOS package for audio native libraries.
+  media_kit_libs_macos_audio: ^1.1.1             # macOS package for audio native libraries.
   media_kit_libs_windows_audio: ^1.0.5           # Windows package for audio native libraries.
   media_kit_libs_linux: ^1.1.0                   # GNU/Linux dependency package.
 ```

